### PR TITLE
Alchemy <> Polygon Docs Changes

### PR DESCRIPTION
**Pull request by nstrike2**
Hi Polygon team and @vidyaakrishna! My name is Neetish, and I'm a Go-To-Market Engineer for Alchemy. I'm aware you've been speaking to Jason Shah and the rest of our team about changes to the Polygon docs regarding Alchemy! We've done our best to provide helpful tutorials and valuable resources for developers looking to build on Polygon.

This PR includes all the changes we've previously discussed. Feel free to comment on the PR or message me at @youngsharm on Twitter/TG with any questions — or if you just wanna jam on crypto! Super happy to partner with you guys.

### CODE REVIEW

File: docs/develop/network-details/network.mdx [hunk 1]
```diff
@@ -31,8 +31,8 @@ The documentation corresponding contains details for the RPC - HTTP, WS and Dagg
...
+| RPC          | `https://rpc-mumbai.matic.today` or <br/>` https://matic-mumbai.chainstacklabs.com` or <br/>`https://rpc-mumbai.maticvigil.com` or <br/>`https://matic-testnet-archive-rpc.bwarelabs.com`  <br/> <br/>Public RPCs may have traffic or rate-limits depending on usage. You can sign up for a dedicated free RPC URL at the following: <br/> <ul><li>[Alchemy](https://www.alchemy.com/)</li><li>[Ankr](https://www.ankr.com/)</li><li>[BlockPI](https://chains.blockpi.io/#/polygon)</li><li> [Chainstack](https://chainstack.com/build-better-with-polygon/)</li><li> [Getblock](https://getblock.io/en/) </li><li>[Infura](https://infura.io)</li><li>[MaticVigil](https://rpc.maticvigil.com/)</li><li>[Moralis](https://moralis.io)</li><li>[Pocket Network](https://www.portal.pokt.network/)</li><li> [QuickNode](https://blog.quicknode.com/polygon-matic-is-now-live-on-quiknode-api/) </li></ul>    |
```
Comment (nstrike2) [inline docs/develop/network-details/network.mdx:34]: Putting resources in alphabetical order!

File: docs/develop/network-details/network.mdx [hunk 2]
```diff
@@ -31,8 +31,8 @@ The documentation corresponding contains details for the RPC - HTTP, WS and Dagg
...
+| Websocket    | `wss://rpc-mumbai.matic.today` or <br/>`wss://ws-matic-mumbai.chainstacklabs.com` or <br/> `wss://rpc-mumbai.maticvigil.com/ws` or <br/> `wss://matic-testnet-archive-ws.bwarelabs.com`  <br/><br/>Public RPCs may have traffic or rate-limits depending on usage. You can sign up for a dedicated free RPC URL at the following: <br/> <ul><li>[Alchemy](https://www.alchemy.com/)</li><li>[Ankr](https://www.ankr.com/)</li><li> [Chainstack](https://chainstack.com/build-better-with-polygon/)</li><li> [Getblock](https://getblock.io/en/) </li><li>[Infura](https://infura.io)</li><li>[MaticVigil](https://rpc.maticvigil.com/)</li><li>[Moralis](https://moralis.io)</li><li> [QuickNode](https://blog.quicknode.com/polygon-matic-is-now-live-on-quiknode-api/) </li></ul>     |
```
Comment (nstrike2) [inline docs/develop/network-details/network.mdx:35]: Putting resources in alphabetical order!

File: docs/develop/network-details/network.mdx [hunk 3]
```diff
@@ -60,8 +60,8 @@ The documentation corresponding contains details for the RPC - HTTP, WS and Dagg
...
.../twitter.com/0xPolygon/status/1431277874238169094) and [here](https://blog.polygon.technology/polygon-rpc-gateway-will-provide-a-free-high-performance-connection-to-the-polygon-pos-blockchain-df148958d418).<br/><br/>Use these alternative RPCs only if the above RPC fails:<br/> `https://rpc-mainnet.matic.network` or <br/>`https://matic-mainnet.chainstacklabs.com` or <br/>`https://rpc-mainnet.maticvigil.com` or <br/>`https://rpc-mainnet.matic.quiknode.pro` or <br/> `https://matic-mainnet-full-rpc.bwarelabs.com`<br/><br/>Public RPCs may have traffic or rate-limits depending on usage. You can sign up for a dedicated free RPC URL at the following: <br/> <ul><li>[Alchemy](https://www.alchemy.com/)</li><li>[Ankr](https://www.ankr.com/)</li><li>[BlockPI](https://chains.blockpi.io/#/polygon)</li><li> [Chainstack](https://chainstack.com/build-better-with-polygon/)</li><li> [Getblock](https://getblock.io/en/) </li><li>[Infura](https://infura.io)</li><li>[MaticVigil](https://rpc.maticvigil.com/)</li><li>[Moralis](https://moralis.io)</li><li>[Pocket Network](https://www.portal.pokt.network/)</li><li> [QuickNode](https://blog.quicknode.com/polygon-matic-is-now-live-on-quiknode-api/) </li></ul>  |
```
Comment (nstrike2) [inline docs/develop/network-details/network.mdx:63]: Putting resources in alphabetical order!

File: docs/develop/network-details/network.mdx [hunk 4]
```diff
@@ -60,8 +60,8 @@ The documentation corresponding contains details for the RPC - HTTP, WS and Dagg
...
+| Websocket    | `wss://rpc-mainnet.matic.network` or <br/>`wss://ws-matic-mainnet.chainstacklabs.com`  or <br/> `wss://rpc-mainnet.maticvigil.com/ws` or <br/>`wss://rpc-mainnet.matic.quiknode.pro` or <br/> `wss://matic-mainnet-full-ws.bwarelabs.com` or <br/>`wss://matic-mainnet-archive-ws.bwarelabs.com`<br/> <br/>Public RPCs may have traffic or rate-limits depending on usage. You can sign up for a dedicated free RPC URL at the following: <br/> <ul><li>[Alchemy](https://www.alchemy.com/)</li><li>[Ankr](https://www.ankr.com/)</li><li> [Chainstack](https://chainstack.com/build-better-with-polygon/)</li><li> [Getblock](https://getblock.io/en/) </li><li>[Infura](https://infura.io)</li><li>[MaticVigil](https://rpc.maticvigil.com/)</li><li>[Moralis](https://moralis.io)</li><li> [QuickNode](https://blog.quicknode.com/polygon-matic-is-now-live-on-quiknode-api/) </li></ul>     |
```
Comment (nstrike2) [inline docs/develop/network-details/network.mdx:64]: Putting resources in alphabetical order!

**Review by vidyaakrishna — APPROVED** [review]

**Comment by nstrike2** [comment]
Hi @vidyaakrishna, given that these changes have been approved, can we finally get this merged?

**Comment by nstrike2** [comment]
Hi @vidyaakrishna, given that these changes have been approved, can we get this merged?

cc: @anurag-arjun

**Pull request merged**